open-banking-accelerator/components/consent-management/com.wso2.openbanking.accelerator.consent.extensions/src/main/java/com/wso2/openbanking/accelerator/consent/extensions/authorize/utils/ConsentRetrievalUtil.java

```diff
@@ -1,12 +1,12 @@
 /**
- * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
- *
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
```

Comment:
Since. this file is not newly added, I think we dont need to change the licence header year right?

Reply:
Fixed in [7207d30]